static/app/views/replays/detail/network/index.tsx

```diff
@@ -296,51 +295,49 @@ const EmptyText = styled(Text)`
 `;
 
 const Item = styled('div')<{
+  hasOccurred: boolean;
+  isCurrent: boolean;
+  isStatusError: boolean;
+  timestampSortDir: SortDirection | undefined;
```

Comment:
turns out all these are always set, so we don't need default params down below
ie: no more:
`color: ${({hasOccurred = true, isStatusError, timestampSort, ...p}) => { ...`

just keep it simple: `color $(p => { ...`